app/assets/stylesheets/active_admin.css.scss

```diff
@@ -22,6 +22,7 @@
 @import "font-awesome";
 @import "active_admin/components/active_admin_scoped_collection_actions";
 @import "vendor/highlightjs/default";
+@import "yeti/yeti.css"
```

Comment:
This might not work in production-mode. Better check in production mode before merge. 

Recently I've had a problem - CSS files would not be imported. In case of such error, the solution is to rename files `*.css` to `*.scss`.

Reply:
thanks. I'll check it